mandatory/2-function-creation.js

```diff
@@ -25,7 +29,10 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  return arrayOfStrings.map(string => string.trim().replace("/" , "").toLowerCase())
```

Comment:
Really good use of the map function! Another way would be to use the for loop, but something like the map function does the same things, uses less lines and is easier to read!